app/models/hyrax/pcdm_collection.rb

```diff
@@ -70,6 +70,15 @@ def permission_manager
       @permission_manager ||= Hyrax::PermissionManager.new(resource: self)
     end
 
+    # Get the collection_type when accessed
+    def collection_type
```

Comment:
we have generally tried to avoid putting new behavior on models.

which tests does this impact? is there a more valkyrie style way of passing them?

Reply:
This is the test that was calling it explicitly. 
https://github.com/samvera/hyrax/blob/917b4568bdedda15bfa3c65d1c456111156764f7/spec/controllers/hyrax/dashboard/nest_collections_controller_spec.rb#L138-L144

There are a few places in the code base where a collection's collection type is accessed through it so I thought it might make sense to move this convenience method here, but I could go apply a new pattern to all places that expect it to exist.
Example:
https://github.com/samvera/hyrax/blob/917b4568bdedda15bfa3c65d1c456111156764f7/app/controllers/concerns/hyrax/collections_controller_behavior.rb#L81-L82

Reply:
I went with `Hyrax::CollectionType.for(collection: collection)` because I felt it was readable and the best option.